stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -914,7 +914,13 @@ internal class ChatDomainImpl internal constructor(
         messageId: String,
         olderMessagesOffset: Int,
         newerMessagesOffset: Int,
-    ): Call<Message> = LoadMessageById(this).invoke(cid, messageId, olderMessagesOffset, newerMessagesOffset)
+    ): Call<Message> {
+        validateCid(cid)
+        val channelController = channel(cid)
```

Comment:
As far as we are returning a `Call<Message>` that represent a deferred computation, all this code should be performed inside of the call when it is run

Reply:
Yep, I agree.

Reply:
@JcMinarro Please check the changes.